File: Exercicio4.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL,"");
    int func, hr;
    float recebe, salario;

    printf("Insira o seu número: ");
    scanf("%d",&func);
    printf("\nQuantas horas você trabalhou esse mês? ");
    scanf("%d",&hr);
    printf("\nQuanto você recebe por hora: ");
    scanf("%f",&recebe);

    salario = recebe*hr;
    printf("\n\nNÚMERO: %d \nSALÁRIO: %2.f\n",func,salario);
    return 0;
}

File: Exercicio4.cs
using System;

class Program {
  public static void Main (string[] args)
        {
            int func, hr;
            float recebe, salario;

            Console.WriteLine("Insira o seu numero: ");
            func = int.Parse(Console.ReadLine());
            Console.WriteLine("Quantas horas voce trabalhou este mes? ");
            hr = int.Parse(Console.ReadLine());
            Console.WriteLine("QUanto voce recebe por hora? ");
            recebe = float.Parse(Console.ReadLine());

            salario = recebe * hr;
            Console.WriteLine("\r\nNumero: " + func + "\r\nSalario: " + salario);

        }
}